File: app/src/main/java/com/tetalab/androidtutorial_astronomy/model/Astronomy.java
package com.tetalab.androidtutorial_astronomy.model;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Astronomy {

    @SerializedName("astronomy")
    @Expose
    private List<Astronomy_> astronomy = null;
    @SerializedName("country")
    @Expose
    private String country;
    @SerializedName("state")
    @Expose
    private String state;
    @SerializedName("city")
    @Expose
    private String city;
    @SerializedName("latitude")
    @Expose
    private Double latitude;
    @SerializedName("longitude")
    @Expose
    private Double longitude;
    @SerializedName("timezone")
    @Expose
    private Integer timezone;

    public List<Astronomy_> getAstronomy() {
        return astronomy;
    }

    public void setAstronomy(List<Astronomy_> astronomy) {
        this.astronomy = astronomy;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public Double getLatitude() {
        return latitude;
    }

    public void setLatitude(Double latitude) {
        this.latitude = latitude;
    }

    public Double getLongitude() {
        return longitude;
    }

    public void setLongitude(Double longitude) {
        this.longitude = longitude;
    }

    public Integer getTimezone() {
        return timezone;
    }

    public void setTimezone(Integer timezone) {
        this.timezone = timezone;
    }

}

File: app/src/main/java/com/tetalab/androidtutorial_astronomy/model/Astronomy_.kt
package com.tetalab.androidtutorial_astronomy.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Astronomy_ {

    @SerializedName("sunrise")
    @Expose
    var sunrise: String? = null
    @SerializedName("sunset")
    @Expose
    var sunset: String? = null
    @SerializedName("moonrise")
    @Expose
    var moonrise: String? = null
    @SerializedName("moonset")
    @Expose
    var moonset: String? = null
    @SerializedName("moonPhase")
    @Expose
    var moonPhase: Double? = null
    @SerializedName("moonPhaseDesc")
    @Expose
    var moonPhaseDesc: String? = null
    @SerializedName("iconName")
    @Expose
    var iconName: String? = null
    @SerializedName("city")
    @Expose
    var city: String? = null
    @SerializedName("latitude")
    @Expose
    var latitude: Double? = null
    @SerializedName("longitude")
    @Expose
    var longitude: Double? = null
    @SerializedName("utcTime")
    @Expose
    var utcTime: String? = null

}